docs/open-source-tizen/tizen-studio/package-file-desc-guide.md

```diff
@@ -70,38 +80,38 @@ build() {
     sleep 3
     cat $SRCDIR/a
 }
-intall() {
+install() {
     // Package Directory Set up
     INSTALL_DIRA=$SRCDIR/package/testGitA.package.$TARGET_OS/data
     mkdir -p $INSTALL_DIRA
     cp $SRCDIR/a $INSTALL_DIRA
 }
 ```
-#### **changelog** file
 
-- Included in package file (zip)
-- Update changelog when new package is deployed. <Version Up required>
+### `changelog` File
+
+This file is included in the package file (`.zip`). When a new package is deployed, the changelog is updated. &lt;Version Up is required&gt;
 
 ```text
  * 1.0.12    // Same as pkginfo.manifest version
-- Titile, Desc ,,
+- Title, Desc ,,
 == Sujin Lee <lee.sujin@samsung.com> 20XX-XX-XX
```

Comment:
Do you really want your own name showing here?
Consider changing it to some dummy name, like "John Do".